README.md

```diff
@@ -253,6 +253,12 @@ interface to get information if the usage of space and index names in requests
 is supported.
 * `Schema` structure no longer implements `SchemaResolver` interface.
 
+#### Client tools changes
+
+* `Op` struct made private.
+* Remove `OpSplice` struct.
+* `Operations.Splice` method now accepts 5 arguments instead of 3.
```

Comment:
This changes more about requests.
Let's also hightlight that Update/Upsert/UpdateAsync... API changed.

Reply:
`README` updated.